src/main/java/wooteco/subway/line/dao/LineDao.java

```diff
@@ -0,0 +1,20 @@
+package wooteco.subway.line.dao;
+
+import java.util.List;
+import java.util.Optional;
+import wooteco.subway.line.Line;
+
+public interface LineDao {
+
+    Optional<Line> findLineByName(String name);
+
+    Line save(Line line);
+
+    List<Line> findAll();
+
+    Optional<Line> findLineById(Long id);
+
+    void removeLine(Long id);
```

Comment:
```suggestion
    void remove(Long id);
```
LineDao니까 메서드에 중복해서 Line을 표시할 필요는 없어보이네요
removeById 같이 어떤 조건으로 삭제하는지 표시해주는 메서드명도 좋습니다.